<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFileEditorv4</class>
 <widget class="QWidget" name="SaveFileEditorv4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>472</width>
        <height>714</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" rowspan="4">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="saveFiles">
           <property name="title">
            <string>Save Files</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="13" column="6">
             <widget class="QCheckBox" name="e6">
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Score Bonus</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Collected Emeralds</string>
              </property>
             </widget>
            </item>
            <item row="13" column="2">
             <widget class="QCheckBox" name="e2">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="7">
             <widget class="QSpinBox" name="score">
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Lives</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="7">
             <widget class="QSpinBox" name="lives">
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Current Stage</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Current Stage is Special Stage</string>
              </property>
             </widget>
            </item>
            <item row="13" column="7">
             <widget class="QCheckBox" name="e7">
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="7">
             <widget class="QComboBox" name="curSave">
              <item>
               <property name="text">
                <string>Save File 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Save File 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Save File 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Save File 4</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="13" column="1">
             <widget class="QCheckBox" name="e1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>CharacterID </string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Current Special Stage</string>
              </property>
             </widget>
            </item>
            <item row="13" column="4">
             <widget class="QCheckBox" name="e4">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Save File</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QCheckBox" name="curStageSpecial">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="9" column="1" colspan="7">
             <widget class="QComboBox" name="curStage">
              <item>
               <property name="text">
                <string>New Save</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Palmtree Panic 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Palmtree Panic 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Palmtree Panic 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Collision Chaos 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Collision Chaos 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Collision Chaos 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tidal Tempest 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tidal Tempest 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tidal Tempest 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quartz Quadrant 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quartz Quadrant 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quartz Quadrant 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wacky Workbench 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wacky Workbench 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wacky Workbench 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Stardust Speedway 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Stardust Speedway 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Stardust Speedway 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Metallic Madness 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Metallic Madness 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Metallic Madness 3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="11" column="1" colspan="7">
             <widget class="QComboBox" name="curSpecialStage">
              <item>
               <property name="text">
                <string>Special Stage 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Special Stage 7</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="13" column="3">
             <widget class="QCheckBox" name="e3">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1" colspan="7">
             <widget class="QSpinBox" name="scoreBonus">
              <property name="minimum">
               <number>-2147483647</number>
              </property>
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item row="13" column="5">
             <widget class="QCheckBox" name="e5">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="7">
             <widget class="QComboBox" name="characterID">
              <item>
               <property name="text">
                <string>Sonic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tails</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Knuckles</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sonic and Tails</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Score</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Continues</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1" colspan="7">
             <widget class="QSpinBox" name="continues">
              <property name="minimum">
               <number>-42000000</number>
              </property>
              <property name="maximum">
               <number>42000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QGroupBox" name="globalSettings">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Global Settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Sound FX Volume</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Music Volume:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QDoubleSpinBox" name="vDpadSize">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="minimum">
               <double>0.500000000000000</double>
              </property>
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Knuckles Unlocked</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Original Controls</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="knuxUnlocked">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="boxArtRegion">
              <item>
               <property name="text">
                <string>JP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>US</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EU</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Virtual DPad Size:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Left Virtual DPad XPos</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Right Virtual DPad XPos</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Box Art Region</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Virtual DPad Opacity</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="sfxVol">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QDoubleSpinBox" name="vDpadOpacity">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="activeFile">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Tails Unlocked</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="tailsUnlocked">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Active File</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="ogCtrls">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Left Virtual DPad YPos</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="musVol">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Right Virtual DPad YPos</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QSpinBox" name="vDPadX1">
              <property name="minimum">
               <number>-42000000</number>
              </property>
              <property name="maximum">
               <number>42000000</number>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QSpinBox" name="vDPadY1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-42000000</number>
              </property>
              <property name="maximum">
               <number>42000000</number>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QSpinBox" name="vDPadX2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-42000000</number>
              </property>
              <property name="maximum">
               <number>42000000</number>
              </property>
             </widget>
            </item>
            <item row="12" column="1">
             <widget class="QSpinBox" name="vDPadY2">
              <property name="minimum">
               <number>-42000000</number>
              </property>
              <property name="maximum">
               <number>42000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
